packages/plugins/src/hardhat/simulate.ts

```diff
@@ -29,8 +29,19 @@ import {
   callWithTimeout,
   fetchExecutionTransactionReceipts,
   convertEthersTransactionReceipt,
+  convertEthersTransactionResponse,
+  SphinxTransactionResponse,
 } from '@sphinx-labs/core'
 import { ethers } from 'ethers'
+import { HardhatEthersProvider } from '@nomicfoundation/hardhat-ethers/internal/hardhat-ethers-provider'
+import pLimit from 'p-limit'
+
+const limit = pLimit(5)
```

Comment:
Is there a reason this is defined globally? Seems like it'd make more sense as a local variable in `fetchTransactionResponses`